docs/pages/interfaces.rst

```diff
@@ -38,36 +38,53 @@ for example, can have one, two or three possible types. See the example below:
 Mappable
 --------
 
-:class:`Mappable <returns.interfaces.mappable.MappableN>` interface help us to
-create our own ``Functor`` like :class:`Result <returns.maybe.Maybe>`.
+Something is considered mappable if we can `map` it using a function, generally
+`map` is a method that accepts a function. An example in this library is
+:class:`Maybe <returns.maybe.Maybe>`, it has the `map` method.
+
+:class:`Mappable <returns.interfaces.mappable.MappableN>`
+interface help us to create our own mappable container like
+:class:`Maybe <returns.maybe.Maybe>`.
 
 .. code:: python
 
   >>> from typing import Any, Callable, TypeVar
 
   >>> from returns.interfaces.mappable import Mappable1
 
-  >>> _FunctorValueType = TypeVar('_FunctorValueType', covariant=True)
-  >>> _NewFunctorValueType = TypeVar('_NewFunctorValueType')
+  >>> _NumberType = TypeVar('_NumberType', covariant=True)
+  >>> _NewNumberType = TypeVar('_NewNumberType')
 
-  >>> class MyFunctor(Mappable1[_FunctorValueType]):
-  ...     _inner_value: _FunctorValueType
+  >>> class Number(Mappable1[_NumberType]):
+  ...     _inner_value: _NumberType
```

Comment:
We can skip `_inner_value` annotation here.

Reply:
Done!